specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1505 @@
+            "in": "path",
+            "name": "analysisIntentName",
+            "description": "The name of the Analysis Intent resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "analysisIntent",
+            "schema": {
+              "$ref": "#/definitions/AnalysisIntent"
+            }
+          },
+          "404": {
+            "description": "analysisIntent Not Found",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AnalysisIntentGet": {
+            "$ref": "./examples/AnalysisIntentGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "AnalysisIntent"
+        ],
+        "summary": "Creates the specific Analysis Intent resource",
+        "operationId": "AnalysisIntentPut",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "networkVerifierName",
+            "description": "The name of the Network Verifier resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "analysisIntentName",
+            "description": "The name of the Analysis Intent resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The Analysis Intent resource object to create/update",
+            "schema": {
+              "$ref": "#/definitions/AnalysisIntent"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "analysisIntent was created",
+            "schema": {
+              "$ref": "#/definitions/AnalysisIntent"
+            }
+          },
+          "404": {
+            "description": "Parent resource not found",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "409": {
+            "description": "analysisIntent already exists and cannot be updated",
+            "schema": {
```

Comment:
Calls in ARM need to be idempotent ans support the RE-PUT semantics. You cannot fail a RE-PUT

Reply:
Removed the error for RE-PUT.